app/pages/index.js

```diff
@@ -97,24 +97,29 @@ const Home = ({randomContributors}) => {
             </div>
             <div className="z-10 px-6 mx-auto space-y-12 text-lg text-center text-white lg:space-y-0 lg:space-x-12 lg:flex lg:text-left max-w-7xl xl:font-mediumxl:text-xl">
               <FeatureIcon icon="layers">
-                “Zero-API” data layer lets you{" "}
-                <strong>import server code directly into your components</strong> instead of having
-                to manually add API endpoints and do client-side fetching and caching.
+                Il data layer "Zero-API" consente di{" "}
+                <strong>importare il codice del server direttamente nei componenti</strong> invece
+                di dover aggiungere manualmente gli endpoint delle API ed eseguire il recupero e la
```

Comment:
Lascerei la parola fetch senza traduzione.

Reply:
Ero indeciso in effetti se mantenerlo